### PR TITLE
Nit: allow "make verify" to pass also with helm3

### PR DESCRIPTION
**Pull request by danielfoehrKn**
**How to categorize this PR?**

/kind bug
/priority normal

**What this PR does / why we need it**:

`make verify`  on the current master throws a chart render error.

```
walk.go:74: found symbolic link in path: /Users/d060239/go/src/github.com/gardener/gardener/charts/seed-bootstrap/charts/loki/charts/utils-templates resolves to /Users/d060239/go/src/github.com/gardener/gardener/charts/utils-templates
walk.go:74: found symbolic link in path: /Users/d060239/go/src/github.com/gardener/gardener/charts/seed-bootstrap/charts/utils-templates resolves to /Users/d060239/go/src/github.com/gardener/gardener/charts/utils-templates
walk.go:74: found symbolic link in path: /Users/d060239/go/src/github.com/gardener/gardener/charts/seed-bootstrap/charts/vpa/charts/application/charts/utils-templates resolves to /Users/d060239/go/src/github.com/gardener/gardener/charts/utils-templates
walk.go:74: found symbolic link in path: /Users/d060239/go/src/github.com/gardener/gardener/charts/seed-bootstrap/charts/vpa/charts/application/values.yaml resolves to /Users/d060239/go/src/github.com/gardener/gardener/charts/seed-bootstrap/charts/vpa/values.yaml
walk.go:74: found symbolic link in path: /Users/d060239/go/src/github.com/gardener/gardener/charts/seed-bootstrap/charts/vpa/charts/runtime/charts/utils-templates resolves to /Users/d060239/go/src/github.com/gardener/gardener/charts/utils-templates
walk.go:74: found symbolic link in path: /Users/d060239/go/src/github.com/gardener/gardener/charts/seed-bootstrap/charts/vpa/charts/runtime/values.yaml resolves to /Users/d060239/go/src/github.com/gardener/gardener/charts/seed-bootstrap/charts/vpa/values.yaml
Error: YAML parse error on seed-bootstrap/charts/vpa/charts/runtime/templates/deployment-admission-controller.yaml: error converting YAML to JSON: yaml: line 9: could not find expected ':'

Use --debug flag to render out invalid YAML
```
**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

@timuthy 
**Release note**:

```improvement operator

```

### CODE REVIEW

**Comment by ialidzhikov** [comment]
`make verify` on the master branch is fine. Also locally:
```
$ make verify
> Check
Executing golangci-lint
Executing go vet
Executing gofmt/goimports
All checks successful
> Check Helm charts
Checking for chart symlink errors
Checking whether all charts can be rendered
All checks successful
> Format
> Test
```

It is also failing after `make install-requirements`?

**Comment by danielfoehrKn** [comment]
Thanks @ialidzhikov , I had helm > v3 installed so downgrading fixed the issue.
Nonetheless, `deploymentLabels` is referenced in the helm chart so should also be added in the values.yaml to be consistent?
When upgrading to helm v3 this will also throw a chart rendering error.

File: charts/seed-bootstrap/charts/vpa/values.yaml [hunk 1]
```diff
@@ -8,6 +8,9 @@ global:
 labels:
   garden.sapcloud.io/role: vpa
 
+deploymentLabels:
+  gardener.cloud/role: seed
```
Comment (ialidzhikov) [inline charts/seed-bootstrap/charts/vpa/values.yaml]: We don't need this. Please adapt the 

```
{{ toYaml .Values.deploymentLabels | indent 4 }}
```

usage with 

```
{{- if .Values.deploymentLabels }}
{{ toYaml .Values.deploymentLabels | indent 4 }}
{{- end }}
```

to make helm3 happy.

**Review by ialidzhikov — CHANGES_REQUESTED** [review]

**Review by ialidzhikov — APPROVED** [review]
Thank you!

/lgtm

**Review by timuthy — APPROVED** [review]
Thank you!

**Pull request merged**